src/ControlSystem/ControlErrors/Size/Info.cpp

```diff
@@ -42,6 +42,14 @@ void Info::pup(PUP::er& p) {
   p | discontinuous_change_has_occurred;
 }
 
+void Info::reset() {
+  suggested_time_scale = 0.0;
```

Comment:
What does the value of 0 mean? Should it be NaN?

Reply:
0 is a sentinel value meaning that the control system simply hasn't suggested a timescale.

Reply:
Sounds like something that should be documented..

Reply:
Ok I documented it and added a commit to make it a better sentinel (nullopt)